.github/workflows/buildAndDeploy.yml

```diff
@@ -56,6 +56,7 @@ jobs:
   
   build_and_test_windows:
     runs-on: windows-2022
+    if: startsWith(github.ref, 'refs/tags/') && !contains(github.ref, 'swo-k8s-collector')
```

Comment:
reverting this to original configuration as it takes ~15mins to run for every commit.